### DISCUSSION TITLE
(ubuntu question) hi-res files not being played at full quality

### DISCUSSION THREAD

**Discussion by wolfgangdub** [strawberrymusicplayer/strawberry#1242] [General]
So I'm using an Fiio Ka3 DAC, and it has a little light on it that shows whether the file I'm currently playing is above 48 kHz or below 48 kHz and when I play some 96 kHz files the light never changes color, I also use PowerAmp on Android and the same hi-res files play at full quality and the color changes like normal!

How exactly can I check if the files are being played at full resolution?

**Comment by jonaski** [comment]
Moving this to discussions.

**Comment by jonaski** [comment]
This depends on which audio setup you got, as there are several, alsa only, pulseaudio, pipewire, jack, etc.
But this isn't specific to strawberry, so it's better to contact ubuntu, alsa, pulseaudio, pipewire, or jack community support.
To avoid resampling to 48khz with ALSA you need to disable dmix, usually done by setting a device such as.: hw:0,0
But most Linux distributions comes with pulseaudio or pipewire pre-installed as the default, either you need to uninstall, disable and stop those, or configure PulseAudio or PipeWire to avoid resampling, I have never used either, so I don't know if it works, but I think at least PulseAudio has a configuration option called "avoid-resampling".
Also make sure to disable the equalizer and loudness normalization such as replay-gain in strawberry since also those converts the audio sample rate and/or format.

**Reply by amarildomoney** [reply]
hello, Wolfgangdub here!

so i disabled Pipewire messed around a bit with Qasmixer and changed the Alsa plugin to hw, and eventually i managed to fix the problem!

now, everytime I play a hi-res song, the light changes color normally.

thank you!
